Gemfile

```diff
@@ -11,8 +11,7 @@ gem 'puma', '~> 4.2.1'
 gem 'therubyracer', platforms: :ruby
 
 # Authentication
-gem 'devise_token_auth', '~> 1.0'
-gem 'omniauth'
+gem 'devise_jwt_controllers', github:'Wolox/devise_jwt_controllers', branch: 'release-0.1.3'
```

Comment:
Why we don't publish this gem?

Reply:
👀

Reply:
[*it **is** published*](http://rubygems.org/gems/devise_jwt_controllers). 🙄